lib/game.rb

```diff
@@ -0,0 +1,86 @@
+require 'pry'
+
+class Game
+  attr_reader :turn, :turn_count, :game_deck
+
+  def initialize(turn)
```

Comment:
We probably don't need to initialize with a turn because a game will have many turns. We can create a new turn every loop rather than starting with just one.